### DISCUSSION TITLE
Support for CBU-PWM4 RGB/WHITE mode?

### DISCUSSION THREAD

**Discussion by stvnjnk** [lkempf/casambi-bt-hass#95] [Ideas]
Hi,

I have a couple of CBU-PWM4 connected to RGBW led strips, using the profile RGB/White, where the is a slider for color/white. Would it be possible to add RGBW support and have a separate slider for the color white in home assistant?

![IMG_0043](https://github.com/user-attachments/assets/d50e18f7-196e-4b32-8b2c-53e1dda86d06)

**Comment by lkempf** [comment]
The light announces RGBW support to Homeassistant and there is a slider for white brightness. Isn't that what you are looking for?

![Screenshot_20241226-105726](https://github.com/user-attachments/assets/8310661a-7e4f-4432-8f11-376a1278fd5a)

**Comment by stvnjnk** [comment]
Hi,

Yes, that is correct, although I do not have the slider on home assistant. The profile of the PWM4 is "RGB/WHITE" and not the "RGBW" one, which is correctly identified by the integration. 

<img width="670" alt="Screenshot 2024-12-26 at 12 45 44" src="https://github.com/user-attachments/assets/357ac331-b793-4af4-a441-840222473f54" />
<img width="609" alt="Screenshot 2024-12-26 at 12 45 49" src="https://github.com/user-attachments/assets/6bdbd1a6-ebee-4272-844b-2d9cca3baab9" />
<img width="605" alt="Screenshot 2024-12-26 at 12 46 07" src="https://github.com/user-attachments/assets/21029fa5-b9b8-4027-a825-3ac3631db123" />
<img width="1184" alt="Screenshot 2024-12-26 at 13 01 53" src="https://github.com/user-attachments/assets/2420eb23-ce77-4f60-adbe-59d030ffc26e" />

**Comment by stvnjnk** [comment]
I also wanted to add something regarding the casambi scenes, is it possible to get the state of the scene, if it is on/off. The reason for this is that all of my casambi switches trigger scenes, and I was thinking of triggering another scene or light in home assistant when pressing a casambi switch (which activates a scene). (btw, thank you for the integration, great work)
